### DISCUSSION TITLE
NetBox In a Docker Container

### DISCUSSION THREAD

**Discussion by jrosario78** [DanSheps/netbox-config-backup#82] [Q&A]
Hi - Do you have any instructions that can help configure this plugin when Netbox running in a container? There are some steps that I am not really sure how to tackle like setting up the service when Netbox is in a container. What is the equivalent of `/etc/systemd/system/ncb-rq@.service` and `ExecStart=/opt/netbox/venv/bin/python3 /opt/netbox/netbox/manage.py rqworker --name ncb@%i --with-scheduler netbox_config_backup.jobs` in a container world? Thanks in advance.

**Comment by DanSheps** [comment]
Unfortunately I don't have experience running this in docker.  Perhaps someone else can respond.
